cirq-google/cirq_google/engine/engine_client_test.py

```diff
@@ -352,7 +353,16 @@ def test_create_job(client_constructor):
     labels = {'hello': 'world'}
     client = EngineClient()
     assert client.create_job(
-        'proj', 'prog', 'job0', ['processor0'], run_context, 10, 'A job', labels
```

Comment:
@verult Should we expect existing tests to pass, or are we okay with a breaking change here?